Adyen.IntegrationTest/BaseTest.cs

```diff
@@ -22,7 +22,7 @@ public class BaseTest
         public PaymentResult CreatePaymentResult()
         {
             var client = CreateApiKeyTestClient();
-            var payment = new PaymentService(client);
+            var payment = new GeneralService(client);
```

Comment:
`GeneralService` doesn't look better than the previous `PaymentService`. How about treating classic PAL as single file service and keep the old name?

Reply:
@michaelpaul, I was looking at doing it that way. However I ran into the problem of not being able to figure out how to squash two or more groups/tags into one file. If you have any idea please let me know